tests/integration/test_keeper_map/test.py

```diff
@@ -98,13 +107,13 @@ def create_drop_loop(index, stop_event):
         if stop_event.is_set():
             return
 
-        node.query(
+        node.query_with_retry(
             f"CREATE TABLE {table_name} (key UInt64, value UInt64) ENGINE = KeeperMap('/test') PRIMARY KEY(key);"
         )
-        node.query(f"INSERT INTO {table_name} VALUES ({index}, {i})")
-        result = node.query(f"SELECT value FROM {table_name} WHERE key = {index}")
+        node.query_with_retry(f"INSERT INTO {table_name} VALUES ({index}, {i})")
+        result = node.query_with_retry(f"SELECT value FROM {table_name} WHERE key = {index}")
         assert result.strip() == str(i)
-        node.query(f"DROP TABLE {table_name} SYNC")
+        node.query_with_retry(f"DROP TABLE {table_name} SYNC")
```

Comment:
@qoega this is trying to fix errors like:
https://s3.amazonaws.com/clickhouse-test-reports/0/d41107a94261feb6e7033c0bfc5bce08755d8813/integration_tests__asan__[1/3].html

I think it's just too many threads querying with client so one of the queries can fail with some unrelated subprocess issue.
Retries should catch those exceptions also and if it's a valid/related problem it will also fail on future retries + we do a sanity check at the end (that all table metadata is cleaned i.e. everything was finished successfully)

Reply:
INSERT is actually UPSERT so it should be okay with retries
I'll add for CREATE/DROP the EXISTS clause